snap/local/helper-go/configure.go

```diff
@@ -20,90 +20,93 @@ package main
 
 import (
 	"fmt"
-	"os"
 	"strings"
 
 	hooks "github.com/canonical/edgex-snap-hooks/v2"
 	"github.com/canonical/edgex-snap-hooks/v2/log"
 	"github.com/canonical/edgex-snap-hooks/v2/options"
-	local "github.com/edgexfoundry/device-rest-go/hooks"
+	"github.com/canonical/edgex-snap-hooks/v2/snapctl"
 )
 
+// ConfToEnv defines mappings from snap config keys to EdgeX environment variable
+// names that are used to override individual device-rest's [Device]  configuration
+// values via a .env file read by the snap service wrapper.
+//
+// The syntax to set a configuration key is:
+//
+// env.<section>.<keyname>
+//
+var ConfToEnv = map[string]string{
+	// [Device]
+	"device.update-last-connected": "DEVICE_UPDATELASTCONNECTED",
+	"device.use-message-bus":       "DEVICE_USEMESSAGEBUS",
+}
+
 var cli *hooks.CtlCli = hooks.NewSnapCtl()
 
-func main() {
+// configure is called by the main function
+func configure() {
 	var debug = false
 	var err error
 	var envJSON string
 
 	status, err := cli.Config("debug")
 	if err != nil {
-		fmt.Println(fmt.Sprintf("edgex-device-rest:configure: can't read value of 'debug': %v", err))
-		os.Exit(1)
+		log.Fatalf("edgex-device-rest:configure: can't read value of 'debug': %v", err)
```

Comment:
'debug' handling is done internally in the log package. It doesn't need to be repeated here.

https://github.com/canonical/edgex-snap-hooks/blob/cbda5ae2568b3acc8376c0194c2556c594e30a30/log/init.go#L20-L25